kyaml/setters2/settersutil/settercreator.go

```diff
@@ -43,6 +44,16 @@ type SetterCreator struct {
 	// live apply/preview. This field is added to the setter definition to record
 	// the package publisher's intent to make the setter required to be set.
 	Required bool
+
+	// Path to openAPI file
+	OpenAPIPath string
+
+	// Path to resources folder
+	ResourcesPath string
```

Comment:
Why are these changes needed?

Reply:
To treat it as a library which can be called from elsewhere apart from cmd/config. Currently, the kyaml libraries for cfg are tightly coupled to cmd/config. This makes it reusable from other places as well. This make this api layer reusable.